libs/langgraph/langgraph/func/__init__.py

```diff
@@ -112,27 +111,8 @@ def entrypoint(
     store: Optional[BaseStore] = None,
 ) -> Callable[[types.FunctionType], Pregel]:
     def _imp(func: types.FunctionType) -> Pregel:
-        if inspect.isgeneratorfunction(func):
```

Comment:
why was this removed?

Reply:
re-adding w/ tests to make sure that all behavior is well defined

Reply:
ah wasn't obvious to me how to implement a state update if it was a generator. users already have a way to stream by requesting `StreamWriter`